argoverse/evaluation/eval_forecasting.py

```diff
@@ -67,6 +67,9 @@ def get_displacement_errors_and_miss_rate(
     """Compute min fde and ade for each sample.
 
     Note: Both min_fde and min_ade values correspond to the trajectory which has minimum fde.
+    The Brier Score is defined here:
```

Comment:
Thanks for adding it here. I missed it in my PR.

Reply:
No problem